backend/src/main/java/saci/application/mappers/AnswerMapper.java

```diff
@@ -0,0 +1,20 @@
+package saci.application.mappers;
+
+import saci.application.model.ScoreRequest.Answer;
+import saci.domain.model.Knowledge;
+
+public class AnswerMapper {
+    public static Knowledge toDomain(Answer answer) {
+        Knowledge domainAnswer = new Knowledge();
+        domainAnswer.setId(answer.getKnowledgeId().longValue());
+        domainAnswer.setWeight(answer.getAnswer());
+        return domainAnswer;
+    }
+
+    public static Answer toModel(Knowledge domainAnswer) {
```

Comment:
This method should be called toResponse (as in controller response), as model exists in both domain and application packages.